spec/models/work_spec.rb

```diff
@@ -173,16 +173,21 @@
 
     let(:fake_s3_service_pre) { stub_s3 }
     let(:fake_s3_service_post) { stub_s3(data: [file1, file2]) }
+    let(:fake_s3_service_embargo) { stub_s3(data: [file1, file2]) }
```

Comment:
We had to update the test to account for the fact that files for embargoed works are now moved to the embargo bucket instead of the post-curation bucket.